lib/configuration.js

```diff
@@ -56,7 +54,7 @@ function extractServiceContext(initConfiguration) {
   };
 
   if (has(initConfiguration, 'serviceContext') &&
-      isObject(initConfiguration.serviceContext)) {
+      isPlainObject(initConfiguration.serviceContext)) {
```

Comment:
@ofrobots it does not. It was just a quick fix to eliminate a false positive where a user could supply an array.